Refactor my code after review

JinHoooooou opened the pull request:
Code Review랑 '클린코드', '자바로 배우는 리팩토링 입문' 책을 보고 echoMyMessage 메소드만 일단 리팩토링 해봤는데 리뷰 부탁드립니다.. 
`175fa6d` 커밋입니다. 

---

echoMyMessage 외 메소드들도 리팩토링 해봤는데 리뷰 부탁드립니다.
최대한 가독성 위주로 리팩토링했습니다.

JinHoooooou committed:
Delete useless file

make useless test code just check my message log

JinHoooooou committed:
Change method name

lower camel case to upper camel case

JinHoooooou committed:
Refactor 'echoMyMessage' method

TaeHyoungKwon commented on slack_kw_info_chatbot/src/main/java/com/slack/chatbot/service/KwInfoBotServiceImpl.java:
```diff
@@ -28,22 +27,17 @@
+		if(isCorrectBotName(messageFromUser, SLACK_BOT_NAME)){
+			String echoMessage = extractMyMessage(messageFromUser, SLACK_BOT_NAME);
+			sendBotMessageToChannel(echoMessage,request.getEvent().getChannel());
+			return true;
```
지금 새롭게 리팩토링 한 코드가 읽기 훨씬 좋음 👍 굳굳

TaeHyoungKwon replied:
내가 자바 코드의 베스트 프랙티스를 잘 모르다 보니, 자바 코드에 대해서 가독성을 제외하고는 따로 코멘트 할 것들이 없음

그러니, 코드워즈 문제 풀고 답보면서, 자바 베스트 프랙티스 코드를 어떻게 짜는지 배워서 활용하면 매우 좋을 듯

무튼 위의 리팩토링한 코드는 매우 훌륭함 😄 앞으로도 저렇게 가독성 좋게 고민많이해서 작성해보시길

JinHoooooou committed:
Refactor "echoMyMessage" method 2

extract constant string

TaeHyoungKwon commented:
Merge Go Go!

JinHoooooou committed:
Refactor 'echoMyMessage' method 3

JinHoooooou committed:
efactor "sendBusInfo" method

rename "sendBusInfo" to "sendBusArriveTime"
add constant number
fix enum class
extract method

JinHoooooou committed:
Refactor "sendKwuNotice" method

extract method
fix enum class

JinHoooooou committed:
Refactor "sendKwuStudyRoomSeat" method

extract method
delete useless method
fix enum class

TaeHyoungKwon commented on slack_kw_info_chatbot/src/main/java/com/slack/chatbot/service/KwInfoBotServiceImpl.java:
```diff
@@ -120,7 +111,10 @@ private String makeBusArriveMessageFormat(Elements busListOfStation) {
+		String tags="";
+		if(url.contains(BusOpenApiUrl.BUS_OPEN_API_BASE_URL.getUrl())) tags = "itemList";
+		else if(url.equals(KwuUrl.KWU_NOTICE_URL)) tags = "li.top-notice";
+		else tags = "tr";
```
개인적으로 옆에쓰는 것보다 아래 내리는게 가독성이 더 좋은듯

혹시 가독성관련해서 책에서 저렇게 하라고 했던거라면, 패스

TaeHyoungKwon commented on slack_kw_info_chatbot/src/main/java/com/slack/chatbot/message/BusOpenApiUrl.java:
```diff
@@ -6,16 +6,12 @@
-	BUS_OPEN_API_BASE_URL("http://ws.bus.go.kr/api/rest/arrive/getArrInfoByRoute"),
-	
+	BUS_OPEN_API_BASE_URL("http://ws.bus.go.kr/api/rest/stationinfo/getStationByUid"),
 	BUS_OPEN_API_SERVICE_KEY("?serviceKey=7jHs%2Bx3TFCBGi9X0WFnQeUXUpCC7KJLJ1aA%2BmM3j8cJau%2B67c22LhWy%2FmkWvLG7m%2BieC4iQay%2FgroMRytDrmzQ%3D%3D"),
```
서비스 키는 버젼 관리에서 빼주셈

TaeHyoungKwon commented:
커밋 보조 설명으로 적지 말고,

앞으론 그 커밋들도 나눠서 작성하셈 아래 커밋은 4개로 나눌 수 있음(현재꺼는 일단 놔두셈)

![image](https://user-images.githubusercontent.com/18538418/75154579-58ae3200-5751-11ea-868a-3a3289c727d5.png)

커밋 단위로 코드리뷰 할 경우에, 커밋이 많더라도 보기좋게 나눠져 있으면 커밋 적은 대신 볼게 많은 것보다 훨씬 좋음

TaeHyoungKwon reviewed (changes requested):
서비스 비 키 빼는거랑 if elif else 옆에다 쓰는것 만 확인하고 이거는 Merge 해도 될듯

가독성은 많이 좋아짐

JinHoooooou committed:
delete authentication key for safe

JinHoooooou committed:
fix typo

JinHoooooou commented:
수정 후 Merge했습니다
